### PR TITLE
update dcgm version for 21.07

### PR DESCRIPTION
**Pull request by jbkyang-nvi**

### CODE REVIEW

**Comment by GuanLuo** [comment]
You should branch out from 21.07, apply the change, and create PR. Now it is more like merging main to 21.07

**Comment by jbkyang-nvi** [comment]
> My bad, the git history was messed up for a short period of time, now it is good.

@GuanLuo no the base was 20.07 for a second. My bad 😓

**Review by deadeyegoodwin — DISMISSED** [review]

File: build.py [hunk 1]
```diff
@@ -65,13 +65,13 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.12.0':
```
Comment (CoderHam) [inline build.py:68]: Are the format changes introduced by format.py?

File: build.py [hunk 2]
```diff
@@ -65,13 +65,13 @@
...
-       '2021.2.200', # ORT OpenVINO
-       '2021.2',     # Standalone OpenVINO
-       '2.2.8')      # DCGM version
+    '2.12.0': (
+        '21.07',  # triton container
+        '21.07',  # upstream container
+        '1.8.0',  # ORT
+        '2021.2.200',  # ORT OpenVINO
+        '2021.2',  # Standalone OpenVINO
+        '2.2.3')  # DCGM version
```
Comment (CoderHam) [inline build.py:74]: We can move forward to 2.2.8 for 21.08 so you can close https://github.com/triton-inference-server/server/pull/3143

**Review by CoderHam — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Review by CoderHam — APPROVED** [review]

**Pull request merged**